neqo-transport/src/recovery.rs

```diff
@@ -209,12 +227,152 @@ impl LossRecoverySpaces {
     }
 }
 
+#[derive(Debug)]
+struct CongestionControl {
+    congestion_window: u64, // = kInitialWindow
+    bytes_in_flight: u64,
+    congestion_recovery_start_time: Option<Instant>,
+    ssthresh: u64,
+}
+
+impl Default for CongestionControl {
+    fn default() -> Self {
+        CongestionControl {
+            congestion_window: INITIAL_WINDOW,
```

Comment:
The initial window might be a function of a variable PMTU.

Reply:
I've opened #349, this can wait until a later PR is my current belief.